tests/unit/cloudfront/test_get_distribution_config.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from awscli.testutils import BaseAWSPreviewCommandParamsTest as \
+    BaseAWSCommandParamsTest
+
+
+class TestListDistributions(BaseAWSCommandParamsTest):
+
+    prefix = 'cloudfront get-distribution-config'
+
+    def test_list_distributions(self):
```

Comment:
Yeah if you do not configure cloudfront to preview in the config and use BaseAWSCommandParamsTest, it fails